src/forge/portablemc_forge/__init__.py

```diff
@@ -182,9 +182,12 @@ def prepare(self):
         if not path.samefile(self.main_dir, path.dirname(self.version.context.libraries_dir)):
             raise ForgeInvalidMainDirectory()
 
+        i = 0
         for possible_version in self.possible_artifact_versions:
             installer_url = f"https://maven.minecraftforge.net/net/minecraftforge/forge/{possible_version}/forge-{possible_version}-installer.jar"
-            self.dl.append(DownloadEntry(installer_url, self.installer_file, name=f"installer:{possible_version}"))
+            self.installer_files.append(path.join(self.version_dir, f"installer-{possible_version}.jar"))
```

Comment:
Here, you could store the full path in a variable and then use it in both DownloadEntry constructor and append, this would avoid using a new variable `i`.